src/routes/index.svelte

```diff
@@ -1,91 +1,55 @@
 <script context="module" lang="ts">
 	import type { Load } from '@sveltejs/kit';
 	export const load: Load = async ({ page, fetch }) => {
-		try {
-			const { query } = page;
-			const res = await fetch(`/games.json?${query.toString()}`);
-			if (res.ok) {
-				const games = await res.json();
-				return {
-					props: {
-						games,
-						query,
-						offline: res.statusText === 'offline'
-					}
-				};
-			}
+		const { query } = page;
+		const res = await fetch(`/spellen.json?${query.toString()}`);
+
+		if (res.ok) {
+			const games = await res.json();
+
 			return {
 				props: {
-					status: res.status,
-					error: await res.json(),
+					games,
 					query
 				}
 			};
-		} catch (error) {
-			return {
-				status: 500,
-				error
-			};
 		}
+
+		const { message } = await res.json();
+
+		return {
+			error: new Error(message),
+			props: {
+				query
+			}
+		};
 	};
 </script>
 
 <script lang="ts">
-	import GameList from '$lib/GameList/GameList.svelte';
-	import GameListFilter from '$lib/GameList/GameListFilter.svelte';
-	import { onMount } from 'svelte';
+	import GameListHighlighted from '$lib/GameList/GameListHighlighted.svelte';
+	import GameListCarousel from '$lib/GameList/GameListCarousel.svelte';
 	import type { Game } from '$lib/games';
-	export let offline;
+
 	export let games: Game[];
-	export let query;
-	onMount(() => {
-		if (!navigator.onLine) offline = true;
-		patchGames();
-	});
-	function patchGames() {
-		Promise.all(
-			games.map((game: Game) => {
-				return caches
-					.open('gamesCache')
-					.then((cache) => {
-						return cache.match(`/games/${game.slug}.json`);
-					})
-					.then((response: Response | undefined) => {
-						if (response) game.offline = true;
-						else game.offline = false;
-						return game;
-					});
-			})
-		)
-			.then((patchGames: Game[]) => (games = patchGames))
-			.then(() => {
-				if (offline) games = games.filter((game) => game.offline);
-			});
-	}
+	const newestGames = games.slice(Math.max(games.length - 4, 0));
 </script>
 
 <header>
-	<h1>John Doe</h1>
-	<h2>Goedemorgen!</h2>
+	<h1>Jungle Gym</h1>
 </header>
 
-{#if !offline}
-	<GameListFilter {query} />
-{/if}
-
-<GameList {games} {offline} />
+<GameListHighlighted />
+<GameListCarousel gamesData={newestGames} carouselTitle="Nieuwe spellen" />
+<GameListCarousel gamesData={games} carouselTitle="Favoriete spellen" />
```

Comment:
Kun je dit in een `<main>` tag zetten?
```suggestion
<main>
<GameListHighlighted />
<GameListCarousel gamesData={newestGames} carouselTitle="Nieuwe spellen" />
<GameListCarousel gamesData={games} carouselTitle="Favoriete spellen" />
<main>
```